modules/server/main.tf

```diff
@@ -113,6 +118,22 @@ data "template_file" "forseti_server_env" {
   }
 }
 
+data "template_file" "forseti_server_run" {
+  template = local.server_run_forseti
+
+  vars = {
+    forseti_home                     = var.forseti_home
+    forseti_server_conf_path         = local.server_conf_path
+    policy_library_home              = var.policy_library_home
+    policy_library_sync_enabled      = var.policy_library_sync_enabled
+    policy_library_repository_branch = var.policy_library_repository_branch
+    policy_library_repository_url    = var.policy_library_repository_url
+    policy_library_sync_git_sync_tag = var.policy_library_sync_git_sync_tag
+    storage_bucket_name              = var.server_gcs_module.forseti-server-storage-bucket
+    forseti_scripts                  = var.forseti_scripts
```

Comment:
All of the vars except for forseti_scripts is already included in the forseti_env.sh script, which is sourced by run_forseti.sh. It's not necessary to pass these vars to this script as well. I think it would be better to also include forseti_scripts as an env var within that same forseti_env.sh script.

Reply:
Done